src/components/Identity/index.js

```diff
@@ -100,6 +149,12 @@ const createIdentity = ({ config, logger, cookie }) => {
     commands: {
       getEcid() {
         return optIn.whenOptedIn().then(getEcid);
+      },
+      setCustomerIDs(options) {
```

Comment:
Let's rename to `setCustomerIds` since we had this conversation last week about using `d` for configs and commands; for everything basically :)